CIP-meta-assets (ERC20-like assets)/README.md

```diff
@@ -0,0 +1,448 @@
+
+- the output sender's amount field is equal to the input one minus `transfer.amount`
+
+- the output receiver's amount field is equal to the input one plus `transfer.amount`
+
+- the sender singned the transaction (included in `ctx.tx.signatories` if a `PPubKeyHash` or included a script input if a `PValidatorHash`)
+
+any additional check can be made based on the `account.state` (implementation specific)
+
+#### `Receive`
+
+- `ownInputs` to be of lenght `2`
+
+- the sender input (the `ownInput` with different `utxoRef` of the `validatingInput`)
+being spent with `Transfer` redeemer
+
+- in the sender input `Transefer` redeemer the `to` field is equal to the `account.credentials` (where `account` is the receiver datum)
+
+> **NOTE**
+>
+> the sender input being an `ownInputs` element and being spent with `Transfer` redeemer 
+> implies that the transfer calculation is running in that validation
+>
+> hence we don't need to perform the same calculation here
+
+#### `ForwardCompatibility`
+
+For the current version this redeemer SHOULD always fail.
+
+The main purpose of the redeemer is to avoid breaking compatibilities for addtional implementation specific redeemers
+
+## Rationale: how does this CIP achieve its goals?
+<!-- The rationale fleshes out the specification by describing what motivated the design and what led to particular design decisions. It should describe alternate designs considered and related work. The rationale should provide evidence of consensus within the community and discuss significant objections or concerns raised during the discussion.
+
+It must also explain how the proposal affects the backward compatibility of existing solutions when applicable. If the proposal responds to a CPS, the 'Rationale' section should explain how it addresses the CPS, and answer any questions that the CPS poses for potential solutions.
+-->
+
+## Path to Active
+
+### Acceptance Criteria
+<!-- Describes what are the acceptance criteria whereby a proposal becomes 'Active' -->
+
+- having at least one instance of the smart contracts described on:
+    - mainnet
+    - preview testnet
+    - preprod testnet
+- having at least 2 different wallets integrating meta asset functionalities, mainly:
+    - displayning balance of a specified meta asset if the user provides the address of the respecive account manager contract
+    - transaction creation with `Transfer` redeemers
+
+### Implementation Plan
+<!-- A plan to meet those criteria. Or `N/A` if not applicable. -->
+
+- [x] [PoC implementation](https://github.com/HarmonicLabs/erc20-like)
+- [x] [showcase transactions](https://github.com/HarmonicLabs/erc20-like)
+- [ ] wallet implementation 
+
+## Copyright
+<!-- The CIP must be explicitly licensed under acceptable copyright terms. -->
+
+[CC-BY-4.0]: https://creativecommons.org/licenses/by/4.0/legalcode
+[Apache-2.0]: http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
```suggestion

This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
Recent edits to the CIP `.github` template make this a little easier to understand.